src/System.Net.Sockets/tests/FunctionalTests/Close.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Net.Sockets.Tests
+{
+    public class CloseTests
+    {
```

Comment:
We usually name the test class using singular grammar and suffix of Test, so this should be "CloseTest".  Also, the name of the file should match and shouldn't be "close.cs".

I know that this library hasn't been consistent about this but that is the standard with the majority of all System.Net code.